src/mds/Server.cc

```diff
@@ -7749,6 +7758,21 @@ void Server::handle_client_rename(MDRequestRef& mdr)
     return;
   }
 
+  if (req->get_data().length()) {
+    auto p = req->get_data().cbegin();
+
+    // xattrs on new inode?
```

Comment:
This is a rename -- is there a new inode?

Reply:
Just copied it and will remove it.